tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+    "include": ["src/**/*"],
+    "exclude": ["node_modules", "dist"],
```

Comment:
This should also be redundant, in the case the assumption described above is true.